server/services/StripePayment.js

```diff
@@ -0,0 +1,87 @@
+const stripe = require('stripe')(process.env.SECRET_KEY);
+
+const stripePayment = async (req, res, next) => {
```

Comment:
You may get rid of `next` function if not used

Reply:
👍🏻Done